recipes/gdal/all/conanfile.py

```diff
@@ -154,8 +154,14 @@ def _build_subfolder(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        # if tools.Version(self.version) < "3.0.0":
+        #     del self.options.with_tiledb
```

Comment:
```suggestion
```

Reply:
@uilianries Why this comment specifically? It will be uncommented as soon as TileDB (https://github.com/TileDB-Inc/TileDB) is in CCI.